web/app/components/search-panel.js

```diff
@@ -17,45 +17,54 @@
 import Ember from 'ember';
 
 const APPLICATION_TYPES = {
-  workflow: "Workflow", job: "Job", application: "Application"
+    workflow: "Workflow", job: "Job", application: "Application"
 };
 
 export default Ember.Component.extend({
 
-  searchQuery: null,
-  selectedType: APPLICATION_TYPES.workflow,
-  applicationTypes: [APPLICATION_TYPES.workflow, APPLICATION_TYPES.job, APPLICATION_TYPES.application],
+    searchQuery: null,
+    selectedType: APPLICATION_TYPES.workflow,
+    applicationTypes: [APPLICATION_TYPES.workflow, APPLICATION_TYPES.job, APPLICATION_TYPES.application],
+    selectedTypeToolTip: "Workflow execution id/url",
 
-  notifications: Ember.inject.service('notification-messages'),
+    notifications: Ember.inject.service('notification-messages'),
 
-  actions: {
-    selected(selectionName) {
-      if (selectionName === "Advanced") {
+    actions: {
+        selected(selectionName) {
+    if (selectionName === "Advanced") {
         // go to advanced search when Advanced is clicked
         this.get('router').transitionTo("search");
-      } else {
+    } else {
         this.set("selectedType", selectionName);
-      }
-    },
+        if(selectionName=="Workflow") {
+            this.set("selectedTypeToolTip","Workflow execution url/id");
+        } else if (selectionName=="Job") {
```

Comment:
I have added description to the text box and tool tips as well. I think this will help in reducing the confusion.